recipes/restbed/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package LANGUAGES CXX)
+
+if(NOT CMAKE_CXX_STANDARD)
+    set(CMAKE_CXX_STANDARD 14)
+endif()
```

Comment:
`CMAKE_CXX_STANDARD` should be set as a cache variable to allow it to be overridden. However, the use of target properties is preferred for setting the C++ standard when there is a minimum.

Reply:
Fixed in 12afe0dc8